### PR TITLE
Fixed error of data type mismatch in loss function

### PR DESCRIPTION
**Pull request by chakkritte**
I have tried to run your code using Pytorch 1.4.0 and found an error related to BCEWithLogitsLoss (), so I've fixed it and it works on PyTorch 1.4.0.

### CODE REVIEW

**Comment by yhlleo** [comment]
Thanks :)

**Comment by YouZiUser** [comment]
老外哥，你夯爆了
